code/__DEFINES/atmospherics.dm

```diff
@@ -12,6 +12,7 @@
 #define MOLES			1
 #define ARCHIVE			2
 #define GAS_META		3
+#define SPECIAL			4

```

Comment:
Calling it "SPECIAL" is almost as useless as just leaving it as 4. Come up with a more descriptive name or at least leave a helpful comment